shared/src/business/entities/EntityConstants.js

```diff
@@ -1228,16 +1232,74 @@ const SECTIONS = sortBy([
   TRIAL_CLERKS_SECTION,
 ]);
 
-const TRIAL_STATUS_TYPES = [
-  'Set for Trial',
-  'Dismissed',
-  'Continued',
-  'Rule 122',
-  'A Basis Reached',
-  'Settled',
-  'Recall',
-  'Taken Under Advisement',
-];
+const TRIAL_STATUS_TYPES = {
+  setForTrial: {
+    deprecated: true,
+    displayOrder: 999,
+    label: 'Set for Trial',
+  },
+  dismissed: {
+    deprecated: true,
+    displayOrder: 999,
```

Comment:
Why are some of these all '999'? Is that to force them to the bottom of the list?